src/render/index.html.eco

```diff
@@ -81,38 +125,40 @@ layout: default
       </div>
     </div>
   </div>
-  <!-- Prototyping -->
-  <div class="feature gray-background">
+  <!-- Microservices 2.0 -->
+  <div class="feature white-background">
     <div class="container">
       <div class="row">
         <div class="col-sm-6">
-          <h2>Prototyping</h2>
+          <h2>Microservices 2.0</h2>
           <p>
-            Knot.x gives you the ability to use simple Mocks. This allows you to expose your sample
-            data directly to pages even if the real service is not available yet, and switch to
-            the real service without any further development work.
+            Knot.x was build with micro-services principles in mind, all modules are are <b>independent</b>
```

Comment:
_independent *OF* one another_
Although I'm not exactly sure this is right. A safer bet would be to say the modules are _not dependend on one another_